packages/apollo/src/commands/service/check.ts

```diff
@@ -287,10 +287,6 @@ export default class ServiceCheck extends ProjectCommand {
                 // Save the output because we're going to use it even if we throw. `runTasks` won't return
                 // anything if we throw.
                 Object.assign(taskOutput, ctx);
-
-                task.title = `Validated local schema against tag ${chalk.blue(
```

Comment:
I think this is still valuable for the tense change

Reply:
Good call.